Add section about SSH_ASKPASS (#98)
radicle-dev/radicle-docs: docs/using-radicle/troubleshooting.md

**Note** that the `git-rad-remote` helper was included with your installation of
**Radicle Upstream** and lives under `$HOME/.radicle/bin`.

### Radicle does not prompt for password
Every git command interacting with the rad remote prompts you for the password. It does so via git which relies on the `SSH_ASKPASS` environment variable.

> If you have that variable broken you wont get prompted for a password and see:

Reviewer: ```suggestion
If you have that variable broken you won't get prompted for a password and see:
```

```
Error: Error unsealing key: Unable to decrypt secret box using the derived key
```

> (Which is the error message for entering a wrong password also.)

Reviewer: ```suggestion
**Note**: this is also the error message for entering the wrong password.
```
